src/object-number.scss

```diff
@@ -0,0 +1,71 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-number+(--success | --warning | --error)
+*/
+$block: #{$fd-namespace}-object-number;
+
+$color-states: (
+  "positive": ("color": var(--sapPositiveTextColor)),
+  "critical": ("color": var(--sapCriticalTextColor)),
+  "negative": ("color": var(--sapNegativeTextColor)),
+  "informative": ("color": var(--sapInformativeTextColor))
+);
+
+.#{$block} {
+  // TEXT VARIABLES
+  $fd-object-number-font-size: var(--sapFontSize) !default;
+  $fd-object-number-neutral: var(--sapNeutralTextColor);
+
+  // LARGE DESIGN VARIABLES
+  $fd-object-number-font-size-large: 1.5rem;
+
+  @include fd-reset();
+
+  color: $fd-object-number-neutral;
+  max-width: 100%;
+  display: inline-block;
+
+  &__text,
+  &__unit {
+    @include fd-reset();
+    @include fd-ellipsis();
+
+    font-size: $fd-object-number-font-size;
```

Comment:
fd-reset mixin is bringing `font-size: var(--sapFontSize);` so you don't need this